backend/pkg/api/data_access/user.go

```diff
@@ -96,13 +143,27 @@ func (d *DataAccessService) UpdatePasswordResetTime(ctx context.Context, userId
 }
 
 func (d *DataAccessService) GetEmailConfirmationHash(ctx context.Context, userId uint64) (string, error) {
-	// TODO @DATA-ACCESS
-	return d.dummy.GetEmailConfirmationHash(ctx, userId)
+	var result string
+
+	err := d.userReader.GetContext(ctx, &result, `
+    	SELECT
+			email_confirmation_hash
+		FROM users
+		WHERE id = $1`, userId)
+
+	return result, err
 }
```

Comment:
I'm honestly not even sure why this method exists, I don't think it's needed. My bad, would you remove it? :)

Reply:
Sure, removed.